### ISSUE TITLE
Ensure on-tap gets indexed somewhere

### ISSUE THREAD

**Issue by arthurevans** [Polymer/docs#1279]
Currently, searching for "on-tap" doesn't yield any useful results in the 1.0 docs.

However, since we use it often in our code samples, people see it any try to find it in the docs. Not knowing that "on-<event>" is a general pattern and "tap" is a polymer gesture event, they expect to find documentation for a specific "on-tap" attribute.

Because it's a FAQ, we should add a note and make sure search sends people to the correct section.

**Comment by addyosmani** [comment]
+1. Through search I found it easier to discover the old 0.5 gestures doc than I did locating https://www.polymer-project.org/1.0/docs/devguide/events.html. 

In addition to an FAQ, is it worth expanding the current events guide to call out `on-tap`, `on-track` etc in text-form somewhere so that it's easier to text search? It's not a particularly lengthy page but a ctrl-f for `on-tap` doesn't bring it up either. Might throw some beginners off.
